Apps/Admin/Tests/Functional/cypress/integration/ui/support.cy.js

```diff
@@ -108,7 +108,7 @@ describe("Support", () => {
         cy.get("[data-testid=user-banner-feedback-warning-message]").should(
             "be.visible"
         );
-        verifySingleSupportResult(hdid, phnDuplicate);
+        verifySingleSupportResult(String(), phnDuplicate);
```

Comment:
Is there any benefit to using `String()` instead of `""`?

Reply:
Changed to "".  No reason.  It looked better than null or "".